s2i/assemble

```diff
@@ -61,7 +61,6 @@ else
 	rm -rf .npm
 fi
 
-DEPLOY_DIR=/opt/app-root/output
```

Comment:
What happens if `DEPLOY_DIR` is not set? Shouldn't there be a default?

Reply:
it is set in the Dockerfile.  although DEPLOY_DIR is just suppose to be an internal thing and not exposed to people.  it was an attempt to not repeat myself writing "/opt/app-root/output", but it is causing confusion,  so perhaps i should just remove it(the variable).  The reason i moved it to the Dockerfile was because variables created and set in the assemble script don't seem to persist to the run script